core/ordering/cosipbft/pbft/mod.go

```diff
@@ -53,6 +55,41 @@ func (s State) String() string {
 	}
 }
 
+// defines prometheus metrics
+var (
+	promBlocks = promauto.NewGauge(prometheus.GaugeOpts{
+		Name: "dela_cosipbft_blocks_total",
+		Help: "total number of blocks",
+	})
+
+	promTxs = promauto.NewHistogram(prometheus.HistogramOpts{
+		Name:    "dela_cosipbft_transactions_block",
+		Help:    "total number of transactions in the last block",
+		Buckets: []float64{0, 1, 2, 3, 5, 8, 13, 20, 30, 50, 100},
+	})
+
+	promTxsTot = promauto.NewCounter(prometheus.CounterOpts{
+		Name: "dela_cosipbft_transactions_tot",
+		Help: "total number of transactions in history",
+	})
+
+	promRejectedTxs = promauto.NewHistogram(prometheus.HistogramOpts{
+		Name:    "dela_cosipbft_transactions_rejected_block",
+		Help:    "total number of rejected transactions in the last block",
+		Buckets: []float64{0, 1, 2, 3, 5, 8, 13, 20, 30, 50, 100},
+	})
+
+	promRejectedTxsTot = promauto.NewCounter(prometheus.CounterOpts{
+		Name: "dela_cosipbft_transactions_rejected_tot",
+		Help: "total number of rejected transactions in history",
+	})
```

Comment:
if you're using a histogram, you don't need a counter for the same values.
Because histograms are implemented as time-series (one per bucket), you can just sum across all buckets to have the total.